packages/contracts/contracts/optimistic-ethereum/OVM/predeploys/OVM_SequencerEntrypoint.sol

```diff
@@ -6,137 +6,56 @@ pragma solidity >0.5.0 <0.8.0;
 import { iOVM_ECDSAContractAccount } from "../../iOVM/accounts/iOVM_ECDSAContractAccount.sol";
 
 /* Library Imports */
-import { Lib_BytesUtils } from "../../libraries/utils/Lib_BytesUtils.sol";
-import { Lib_OVMCodec } from "../../libraries/codec/Lib_OVMCodec.sol";
-import { Lib_ECDSAUtils } from "../../libraries/utils/Lib_ECDSAUtils.sol";
+import { Lib_EIP155Tx } from "../../libraries/codec/Lib_EIP155Tx.sol";
 import { Lib_ExecutionManagerWrapper } from "../../libraries/wrappers/Lib_ExecutionManagerWrapper.sol";
 
 /**
  * @title OVM_SequencerEntrypoint
  * @dev The Sequencer Entrypoint is a predeploy which, despite its name, can in fact be called by 
  * any account. It accepts a more efficient compressed calldata format, which it decompresses and 
  * encodes to the standard EIP155 transaction format.
- * This contract is the implementation referenced by the Proxy Sequencer Entrypoint, thus enabling
- * the Optimism team to upgrade the decompression of calldata from the Sequencer.
  * 
  * Compiler used: optimistic-solc
  * Runtime target: OVM
  */
 contract OVM_SequencerEntrypoint {
-
-    /*********
-     * Enums *
-     *********/
-    
-    enum TransactionType {
-        NATIVE_ETH_TRANSACTION,
-        ETH_SIGNED_MESSAGE
-    }
+    using Lib_EIP155Tx for Lib_EIP155Tx.EIP155Tx;
```

Comment:
consistency nit: this PR is I think the first time the codebase where we do the `using ... for` syntax.  Not super opinionated that this shouldn't be done but maybe we make a ticket to use this more often if there are other places this could be done.

Reply:
We're using it in a few other places but not frequently. I agree that we could consider using this more often.